LibraBFT/ImplShared/Interface/Output.agda

```diff
@@ -64,6 +64,14 @@ module LibraBFT.ImplShared.Interface.Output where
   IsLogErr (SendBRP _ _)         = ⊥
   IsLogErr (SendVote _ _)        = ⊥
 
+  LogErrMB : Output → Maybe ErrLog
```

Comment:
Why the capital `L`?

Reply:
No (good :smile:) reason.  I'll fix it before merging.  Thanks.

Reply:
Same for `I` in `IsLogErr`

Reply:
I disagree about `IsLogErr`.  Yes, `IsLogErr` is a function, but `IsLogErr x` is a set/type, and that's the way it's generally used, and is consistent with other similar things nearby, and makes intuitive sense to me.